Functions.Templates/Templates/SqlInputBinding-CSharp/.template.config/vs-2017.3.host.json

```diff
@@ -0,0 +1,38 @@
+{
+  "name": {
+    "text": "SQL input binding",
+    "package": "{E9BCFE2B-304A-4D7F-836F-F3A4FB1A1451}",
+    "id": "3000"
+  },
+  "description": {
+    "text": "A C# function that takes a SQL query to run on a provided table or view and returns the output of the query",
+    "package": "{E9BCFE2B-304A-4D7F-836F-F3A4FB1A1451}",
+    "id": "3001"
+  },
+  "order": 400,
+  "icon": "vs-2017.3/functionF.png",
+  "learnMoreLink": "https://docs.microsoft.com/en-us/azure/azure-functions/functions-bindings-azure-sql-input?tabs=csharp",
```

Comment:
1. Links should not contain the lcid (en-us)
2. We should probably make an aka.ms link for these so we can update them without needing to push a new template update if we want

Reply:
Hm are we okay with using the aka.ms/sqlbindings link or should I create new ones for the input/output binding pages?

Reply:
Let's create new ones (aka.ms/sqlbindingsinput and aka.ms/sqlbindingsoutput)